docs/book/src/getting-started.md

```diff
@@ -146,13 +146,13 @@ reconcile App {
     return reconcile.Result{}, err
   }
 
-  // Check if a Service for the app exists, if not, create one 
+  // Check if a Service for the app exists, if not, create one
   // If there's an error, then restart from the beginning of the reconcile
   if err != nil {
     return reconcile.Result{}, err
   }
 
-  // Look for Database CR/CRD 
+  // Look for Database CR/CRD
```

Comment:
those changes make no part of this PR please rebase with master and if it still 
revert